service/collector.go

```diff
@@ -255,11 +251,6 @@ func (col *Collector) shutdownServiceAndTelemetry(ctx context.Context) error {
 		errs = multierr.Append(errs, fmt.Errorf("failed to shutdown service after error: %w", err))
 	}
 
-	// TODO: Move this as part of the service shutdown.
-	// shutdown telemetryInitializer
-	if err := col.telemetry.shutdown(); err != nil {
-		errs = multierr.Append(errs, fmt.Errorf("failed to shutdown collector telemetry: %w", err))
-	}
 	return errs
 }
```

Comment:
The `shutdownServiceAndTelemetry` func can probably be removed now

Reply:
Done.